app/services/chat_publisher.rb

```diff
@@ -195,6 +195,14 @@ def self.publish_channel_status(chat_channel)
     )
   end
 
+  def self.publish_chat_channel_metadata(chat_channel)
```

Comment:
Perhaps in future we will have more to add here to make this more useful

Reply:
Also interested to hear about this approach, I didn't think doing a joined/left channel event would be necessary (too noisy) and multiple people can join/leave at once with auto join channel so I didn't want to send tons of these through either. Member list only really needs to be updated in one place and DM channels cannot change members currently either.